.test-infra/jenkins/JobBuilder.groovy

```diff
@@ -42,7 +42,7 @@ class JobBuilder {
                             jobDefinition = {}) {
     JobBuilder jb = new JobBuilder(scope, jobDefinition)
     jb.defineAutoPostCommitJob(nameBase)
-    jb.defineGhprbTriggeredJob(nameBase + "_hgprb", triggerPhrase, githubUiHint, false)
+    jb.defineGhprbTriggeredJob(nameBase + "_PullRequest", triggerPhrase, githubUiHint, false)
```

Comment:
There have been issues where long job names cause test failures, how about _PR as the suffix?